Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -4410,6 +4423,90 @@ script:
     description: Creates a WildFire analysis best practice profile.
     execution: false
     name: panorama-create-wildfire-best-practice-profile
+  - arguments:
+    - default: false
+      description: The template to use when running the command. Overrides the template
+        parameter (Panorama instances). If not given, will use the integration paramater.
+      isArray: false
+      name: template
+      required: false
+      secret: false
+    - default: false
+      description: The template stack to use when running the command.
+      isArray: false
+      name: template_stack
+      required: false
+      secret: false
+    - default: false
+      defaultValue: vsys1
```

Comment:
should not have default values so it will use the param

Reply:
It should only use the param if it exists and this argument is not entered as I checked with @jtschichold and it seems firewall instances should be able to work with multiple vsys.

it should default to `vsys1` if neither the argument and the params are not entered.
I think the implementation is incorrect right now, I'll fix that.